db/people.yml

```diff
@@ -61,6 +61,101 @@ achoudhry:
       status: undergrad
   name: Akshay Choudhry
   position: Undergraduate Researcher
+aday:
+  active: false
+  aka: []
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: bio
+  education:
+    - begin_year: 0
+      institution: fix.me
+      degree: fix.me
+  employment:
+    - begin_year: 1999
+      position: Visiting Researcher
+      organization: msu
+      advisor: sbillinge
+      group: bg
+  name: A. R. Day
```

Comment:
This is A. Roy Day aka anthony roy day.  He is at John Carrol University.  Not sure if you could find him with this info

Reply:
I did not find info, what should I do?

Reply:
subtract ~4 years for PhD?